flake.nix

```diff
@@ -29,6 +29,10 @@
       url = "github:dfinity/ic";
       flake = false;
     };
+    pocket-ic-src = {
+      url = "github:dfinity/ic/master";
+      flake = false;
+    };
```

Comment:
Why can't you take the `ic-src` for this? Or was that used for something else and you want to keep the `pocket-ic-src` separate from that?

Reply:
thanks Bas for noticing it! I wanted to keep pocket-ic stuff separate from drun and initially it seemed to make sense, now perhaps not; let me think about it some more.

Reply:
Makes sense. It's also easy to unify this later.

Reply:
We'll also remove drun for good, so it's easy to kick it out then.